docs/gatsby-config.js

```diff
@@ -33,12 +33,12 @@ module.exports = {
     {
       resolve: `@gatsby-contrib/gatsby-plugin-elasticlunr-search`,
       options: {
-        fields: ["title", "excerpt", "slug"],
+        fields: ["title", "excerpt", "path"],
```

Comment:
If there's not a field named `slug`, plugins such as [gatsby-redirect-from](https://github.com/kremalicious/gatsby-redirect-from) won't work, I believe. It assumes there's a field known as `slug`. The benefit of that plugin in my opinion outweighs a slight confusion between `slug` and `fields.slug`.

Reply:
Great catch! I fixed this by renaming `fields.path` to `fields.slug`. I didn't rename anything else, `path` above, because it's not necessary, this `fields` option just has an unfortunate name, it's unrelated to `fields` in GraphQL query.